src/targets/axis_2400.ini

```diff
@@ -0,0 +1,49 @@
+# ********************************
+# Transmitter targets
+# ********************************
+[env:AXIS_THOR_2400_TX_via_UART]
+extends = env_common_esp32
+build_flags =
+	${env_common_esp32.build_flags}
+	${common_env_data.build_flags_tx}
+	-include target/AXIS_THOR_2400_TX.h
+	-D TARGET_AXIS_THOR_2400_TX=1
+	-D USER_SETUP_LOADED=1
+	-D CONFIG_DISABLE_HAL_LOCKS=1
+	-include $PROJECT_DIR/lib/SCREEN/TFT_eSPI_User_Setup.h
```

Comment:
Including header file with `$PROJECT_DIR` present fails to build firmware in unix environment. 

My suggestion is to use:


```
-include lib/SCREEN/TFT_eSPI_User_Setup.h
```

Reply:
fixed at commit 45fefe8